temperature/models.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+from sqlalchemy import Column, Integer, DateTime, Float, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBTemperature(Base):
```

Comment:
In FastAPI, we don't name it DBTemperature; instead, we use models.Temperature or schemas.Temperature.